docs/tutorials/deploy-model.md

```diff
@@ -0,0 +1,246 @@
+
+```python
+from docarray import DocumentArray
+from jina import Executor, requests
+import numpy as np
+
+class TextToImage(Executor):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        import torch
+        from diffusers import StableDiffusionPipeline
+
+        self.pipe = StableDiffusionPipeline.from_pretrained(
+            "CompVis/stable-diffusion-v1-4", torch_dtype=torch.float16
+        ).to("cuda")
+
+    @requests
+    def generate_image(self, docs: DocumentArray, **kwargs):
+    	# image here is in PIL format
+        images = self.pipe(docs.texts).images
+        
+        for i, doc in enumerate(docs):
+            doc.tensor = np.array(images[i])
+```
+
+### Imports
+
+```python
+from docarray import DocumentArray
+```
+
+[Documents](https://docs.docarray.org/fundamentals/document/) and [DocumentArrays](https://docarray.jina.ai/fundamentals/documentarray/) (from the DocArray package) are Jina's native IO format.
+
+```python
+from jina import Executor, requests
+```
+
+Jina's Executor class and requests decorator - we'll jump into these in the next section.
+
+```python
+import numpy as np
+```
+
+In our case, [NumPy](https://numpy.org/) is specific to this Executor only. We won't really cover it in this article, since we want to keep this as a general overview. (And there’s plenty of information about NumPy out there already).
+
+### Executor class
+
+```python
+class TextToImage(Executor):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        import torch
+        from diffusers import StableDiffusionPipeline
+
+        self.pipe = StableDiffusionPipeline.from_pretrained(
+            "CompVis/stable-diffusion-v1-4", torch_dtype=torch.float16
+        ).to("cuda")
+```
+
+All Executors are created from Jina's Executor class. User-definable parameters (like `self.pipe`) are {ref}`arguments <executor-constructor> defined in the `__init__()` method.
+
+### Requests decorator
+
+```python
+@requests
+def generate_image(self, docs: DocumentArray, **kwargs):
+    # image here is in PIL format
+    images = self.pipe(docs.texts).images
+
+    for i, doc in enumerate(docs):
+        doc.tensor = np.array(images[i])
+```
+
+Any Executor methods decorated with `@requests` can be called via an {ref}`endpoint <exec-endpoint>` when the Executor is run or deployed. Since we're using a bare `@requests` (rather than say `@requests(on='/foo')`), the `generate_image()` method will be called as the default fallback handler for any endpoint.
+
+## Deployment: Deploy the Executor
+
+With a Deployment you can run and scale up your Executor, adding sharding, replicas and dynamic batching. 
+
+![](images/deployment.png)
+
+We can deploy our Executor with either the Python API or YAML:
+
+````{tab} Python
+In `deployment.py`:
+
+```python
+from jina import Deployment
+
+dep = Deployment(uses=TextToImage, timeout_ready=-1, install_requirements=True)
```

Comment:
I don't think install_requirements is needed here

Reply:
Fixed in [`3ef7caa` (#5766)](https://github.com/jina-ai/jina/pull/5766/commits/3ef7caa197f8a1ebc253e99a69d4628b417b9aa4)